chia/wallet/puzzles/nft_ownership_transfer_program_one_way_claim_with_royalties_new.clvm

```diff
@@ -0,0 +1,114 @@
+(mod
+  (
+    P2_DELEGATED_PUZZLE_OR_HIDDEN_PUZZLE_MOD_HASH
+    SINGLETON_STRUCT
+    ROYALTY_ADDRESS
+    TRADE_PRICE_PERCENTAGE
+    SETTLEMENT_MOD_HASH
+    CAT_MOD_HASH
+    Current_Owner  ; Truth
+    conditions  ; Truth
+    (new_owner trade_price new_pk tp_solution)  ; solution - created from NFT's innerpuz
+  )
+
+  ; This is a transfer program - which must return (new_owner, Optional[new_transfer_program], conditions)
+
+  (include condition_codes.clvm)
+  (include curry-and-treehash.clinc)
+
+  (defconstant NEW_OWNER -10)
```

Comment:
These constants are not used in this file?

Reply:
yep, thanks